roles/parse_operator_bundle/tasks/main.yml

```diff
@@ -89,39 +89,42 @@
       crd_paths: "{{ crd_paths + [item.path] }}"
     with_items: "{{ crd_paths_result['files'] }}"
 
-  - name: "Include the CSV path into the ansible vars"
-    include_vars:
-      file: "{{ csv_path }}"
-      name: operator_vars
+  - name: "Read csv.yaml file"
+    shell: "cat {{ csv_path }}"
+    register: csv_data
+
+  - name: "Set facts for csv_data"
+    set_fact:
+      csv_vars: "{{ csv_data.stdout | from_yaml }}"
 
   - name: "Determine and set fact for operator specific information - name, pod name, container name and capabilities"
     set_fact:
-      current_csv: "{{ operator_vars.metadata.name }}"
-      operator_pod_name: "{{ operator_vars.spec.install.spec.deployments[0].name }}"
-      operator_container_name: "{{ operator_vars.spec.install.spec.deployments[0].spec.template.spec.containers[0].name }}"
-      operator_capabilities: "{{ operator_vars.metadata.annotations.capabilities }}"
+      current_csv: "{{ (csv_vars | from_yaml).metadata.name }}"
+      operator_pod_name: "{{ (csv_vars | from_yaml).spec.install.spec.deployments[0].name }}"
+      operator_container_name: "{{ (csv_vars | from_yaml).spec.install.spec.deployments[0].spec.template.spec.containers[0].name }}"
+      operator_capabilities: "{{ (csv_vars | from_yaml).metadata.annotations.capabilities }}"
```

Comment:
Do we have to convert the csv_vars again here when it is already done on line#98 ?

Reply:
Same case as mentioned above.